### PR TITLE
Adding the ability to write es6

### PR DESCRIPTION
**Pull request by Guntrisoft**
- adding package.json to install babel and eslint
- running npm install on travis
- enable sprockets-es6
- adding basic bower.json

### CODE REVIEW

**Comment by Guntrisoft** [comment]
@benbarnett

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "telephone_appointment_planner",
+  "private": true,
+  "dependencies": {
+
```
Comment (benlovell) [inline bower.json]: hurrrrr

Comment (benbarnett) [inline bower.json]: can you stop breathing on pull requests please

Comment (Guntrisoft) [inline bower.json]: I was in two minds on whether to add the bower stuff at this point when we don't have any dependencies at this moment, but we will eventually have full calendar, list.js etc etc

**Comment by AndrewVos** [comment]
Merge it!

**Pull request merged**